src/unix.rs

```diff
@@ -0,0 +1,8 @@
+
+/// Unix specific extension methods.
```

Comment:
nit: Do you think we should mirror the docs on the [`DirEntryExt`](https://doc.rust-lang.org/std/os/unix/fs/trait.DirEntryExt.html) trait in the standard library?